src/functions/listFunctions.ts

```diff
@@ -0,0 +1,21 @@
+import Table = require("cli-table");
+import * as backend from "../deploy/functions/backend";
+import { logger } from "../logger";
+import { previews } from "../previews";
+import { Context } from "../deploy/functions/args";
+
+/**
+ * Lists all functions of the Firebase project in order
+ * @param context the Context of the project
+ * @returns a mapping that contains an array of {@link FunctionSpec} in order under the 'functions' key
+ */
+export async function listFunctions(
+  context: Context
+): Promise<{ functions: backend.FunctionSpec[] }> {
+  const bkend = await backend.existingBackend(context, true);
+  const functionSpecs = previews.functionsv2
```

Comment:
nit* `backend.existingBackend` already includes logic for filtering out gcfv2 functions, so this is a redundant check.

Reply:
Removed